templates/admin/tool_shed_repository/deactivate_or_uninstall_repository.mako

```diff
@@ -172,7 +172,7 @@ else:
                                         owner = containing_repository.owner
                                     %>
                                     <li>
-                                        Version <b>${version | h}</b> of ${type | h} <b>${name | h}</b> contained in revision 
+                                        Version <b>${version | h}</b> of ${type | h} <b>${name | h}</b> contained in revision
```

Comment:
this change might actually concatenate the strings - the space seems to be 'formatting' :/

Reply:
I just checked, it does not concatenate.